app/controllers/api/v1/customers_controller.rb

```diff
@@ -6,31 +6,28 @@ module Api
   module V1
     class CustomersController < Api::V1::BaseController
       include AddressTransformation
+      include Api::V1::QueryParam
```

Comment:
Extracted query param validation logic to included module `Api::V1::QueryParam`
I made it as simple as I could + added spec

Reply:
Good one. :+1: